mail_to/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+{
+    "name": """Label on the message about recipients""",
+    "summary": """Add recipient label on the right of message in the Inbox channel""",
```

Comment:
"summary": """Allows you be sure, that all discussion participants were notified""",